MCprep_addon/spawner/spawn_util.py

```diff
@@ -20,13 +20,21 @@
 import re
 
 import bpy
+from bpy.types import (
+  Context, Object, Collection,
+  BlendDataLibraries
+)
```

Comment:
Like mentioned earlier
```suggestion
from bpy.types import (
  Context, Collection, BlendDataLibraries
)
```